config/peerpods/podvm/azure-podvm-image-handler.sh

```diff
@@ -443,6 +443,88 @@ function create_azure_image_from_scratch() {
     echo "Azure image created successfully from scratch"
 }
 
+function create_signed_image_version()
```

Comment:
IIUC, this method isn't signing the image, but "just" registering a pre-signed one.

If that's correct, maybe consider renaming the method and adjusting the comments/debug messages to reflect that more clearly, something like `register_signed_image_version`?

Reply:
Yes, this create refer to creating the image version, it's based on the previous function name if this is confusing i can change